app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -2,24 +2,52 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/**
+ * Mixin to provide experiment related functionality.
+ *
+ * Expects the view to expose:
+ * - either this.getAccount() or this._account
+ * - this.metrics
+ * - this.translator
+ * - this.window
+ *
+ * @mixin ExperimentMixin
+ */
 define(function (require, exports, module) {
   'use strict';
 
+  const { isFunction } = require('underscore');
   const ExperimentInterface = require('lib/experiment');
 
+  function getAccount () {
+    // make no assumptions about the availability of this.getAccount.
+    return (isFunction(this.getAccount) && this.getAccount()) || this._account;
```

Comment:
I don't know enough about the content server to know this, sorry, but presumably it's never possible for `this` to be something that has neither `getAccount` nor `_account`?